helm/chaos-mesh/templates/chaos-dashboard-deployment.yaml

```diff
@@ -48,6 +48,18 @@ spec:
             - name: TZ
               value: {{ .Values.timezone | default "UTC" }}
             {{- end }}
+            - name: TARGET_NAMESPACE
+              value: {{ .Values.controllerManager.targetNamespace }}
+            - name: CLUSTER_SCOPED
+              value: "{{ .Values.clusterScoped }}"
+            {{- if .Values.controllerManager.allowedNamespaces }}
+            - name: ALLOWED_NAMESPACES
+              value: {{ .Values.controllerManager.allowedNamespaces }}
+            {{- end }}
+            {{- if .Values.controllerManager.ignoredNamespaces }}
+            - name: IGNORED_NAMESPACES
+              value: {{ .Values.controllerManager.ignoredNamespaces }}
+            {{- end }}
```

Comment:
seems that `ALLOWED_NAMESPACES` and `IGNORED_NAMESPACES` is useless

Reply:
These ENV will  inject into `common.ControllerCfg` which was dependent in `pkg/utils/selector.go`.

Many API in `pkg/apiserver/common/common.go` use `pkgutils.SelectPods` which located in `pkg/utils/selector.go`. For keeping same action, I think we should set these ENVs.

This is a bad-smell  which already pointed by https://github.com/chaos-mesh/chaos-mesh/issues/1072 😥.